lib/utils/CSVDatabase.cpp

```diff
@@ -174,7 +174,6 @@ CSVDatabase::getIntegerVector(
     if (!append) data.clear();
 
     std::ifstream d_fs(file_name.c_str());
```

Comment:
Why should this line be removed?

Reply:
These lines should be removed to allow the possibility of empty files. In fact, taking `local_tw_csv` as an example, the spatial index and indicator index files read by `getIntegerVector` are optional. If those lines are kept, the run will be terminated when the files are not present, which is not the intention of these routines.